### PR TITLE
Add EventsWidget

### PR DESCRIPTION
**Pull request by fhlavac**
[RHCLOUD-31201](https://issues.redhat.com/browse/RHCLOUD-31201)

Added a new widget displaying notification events in a tabular layout

image

image

### CODE REVIEW

**Comment by epwinchell** [comment]
@fhlavac Looks great, but we've run into our first styling hitch -
cc @Hyperkid123 

Mobile
Screenshot 2024-03-08 at 12 30 48

Single Column
Screenshot 2024-03-08 at 12 31 13

The Table switches to mobile layout based on the viewport size, but that doesn't work for a single column widget  with a large viewport width (it is unaware of the widget width.) Is there any way we can  manipulate the mobile breakpoint - perhaps based on `@container` ?

**Comment by Hyperkid123** [comment]
@epwinchell let's use the container for now. The important part is to get MVP out of the door ASAP. We can tweak the layout and its styling once we have something out of the door.

File: src/Components/DnDLayout/GridTile.tsx [hunk 1]
```diff
@@ -162,7 +162,7 @@ const GridTile = ({ widgetType, title, isDragging, setIsDragging, setWidgetAttri
         </CardTitle>
       </CardHeader>
       <Divider />
-      <CardBody>
+      <CardBody className="pf-v5-u-p-0">
```
Comment (fhlavac) [inline src/Components/DnDLayout/GridTile.tsx:165]: @florkbr @epwinchell do we want the card body padding to be there by default and handle by each widget separately? I removed it for the table to look properly, but we can decide for a different approach

Comment (epwinchell) [reply src/Components/DnDLayout/GridTile.tsx:165]: @fhlavac I removed it on master

**Review by epwinchell — APPROVED** [review]

**Comment by fhlavac** [comment]
Closing and moving the widget to the notifications-ui

**Comment by epwinchell** [comment]
Keeping for testing

**Pull request closed without merging**